projects/sandbox/src/app/cookie-policy/cookie-policy.component.html

```diff
@@ -1,11 +1,12 @@
 <div *ngIf="showing" class="cookie-policy">
   <h2 class="page-title" data-e2e="cookie-policy">
-    Cookie Policy
+    Cookies Policy
   </h2>
 
   <p>
     This cookies policy applies to the Europeana website
```

Comment:
Instead of `Europeana` website should be `sandbox` website or `metis sandbox` website?
![afbeelding](https://github.com/europeana/metis-ui/assets/19560266/8853bb0e-b009-4e69-b9ac-87f86f301ed2)

Reply:
You may be right but we can't remove Europeana!

Reply:
Oh I see, it depends on how we read this. this later in production will look like
`This cookies policy applies to the Europeana website https://metis-sandbox.europeana.eu` I agree to leave it like this then.

Is another way of looking at it.

`This cookies policy applies to the metis sandbox website https://metis-sandbox.europeana.eu`